dockerfiles/Dockerfile-build-crypt

```diff
@@ -0,0 +1,145 @@
+# Build libsodium
+FROM emscripten/emsdk:latest AS SODIUM
+
+RUN wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable.tar.gz \
+    && tar xvf libsodium-1.0.18-stable.tar.gz && cd libsodium-stable \
+    && dist-build/emscripten.sh --sumo
+
+# Build openssl
+FROM emscripten/emsdk:latest AS OPENSSL
+
+RUN wget https://www.openssl.org/source/openssl-1.1.1l.tar.gz \
```

Comment:
shall we try the 1.1.1m version https://www.openssl.org/source/ ?

Reply:
I'll try that, all 1.1.1 versions should be compatible so I don't expect issues with the change